### ISSUE TITLE
Jersey support

### ISSUE THREAD

**Issue by robfromboulder** [resurfaceio/logger-java#55]
https://jersey.github.io/
https://www.baeldung.com/jersey-filters-interceptors

**Comment by robfromboulder** [comment]
Included in v1.9.1

**Comment by robfromboulder** [comment]
Helpful example:  https://stackoverflow.com/questions/33666406/logging-request-and-response-in-one-place-with-jax-rs

Regarding annotations:  https://docs.oracle.com/cd/E19776-01/820-4867/ghrpv/index.html

For local testing:
```
echo 'asdf1234' | curl -X POST -H "Content-Type: text/plain" -H "Cookie: PHPSESSID=298zf09hf012fh2;" -H "Cookie: abra=cadabra" -d @- http://localhost:8080/base/helloworld
```
